std/json.d

```diff
@@ -651,13 +695,13 @@ Params:
     maxDepth = maximum depth of nesting allowed, -1 disables depth checking
     options = enable decoding string representations of NaN/Inf as float values
 */
-JSONValue parseJSON(T)(T json, int maxDepth = -1, JSONOptions options = JSONOptions.none)
+JSONValue parseJSON(T)(T json, int maxDepth = -1, JSONOptions options = JSONOptions.none) @safe
```

Comment:
That's what a `@safe` unittest is for: it will fail to compile if the function itself introduces non-`@safe` operations apart from its arguments.